pkgs/applications/editors/vscode/vscode-oss/product.nix

```diff
@@ -0,0 +1,408 @@
+{ lib }:
+{
+  nameShort = "Code - OSS";
+  nameLong = "Code - OSS";
+  applicationName = "code-oss";
+  dataFolderName = ".vscode-oss";
+  win32MutexName = "vscodeoss";
+  licenseName = "MIT";
+  licenseUrl = "https://github.com/microsoft/vscode/blob/main/LICENSE.txt";
+  serverLicenseUrl = "https://github.com/microsoft/vscode/blob/main/LICENSE.txt";
+  serverGreeting = [ ];
+  serverLicense = [ ];
+  serverLicensePrompt = "";
+  serverApplicationName = "code-server-oss";
+  serverDataFolderName = ".vscode-server-oss";
+  tunnelApplicationName = "code-tunnel-oss";
+  win32DirName = "Microsoft Code OSS";
+  win32NameVersion = "Microsoft Code OSS";
+  win32RegValueName = "CodeOSS";
+  win32x64AppId = "{{D77B7E06-80BA-4137-BCF4-654B95CCEBC5}";
+  win32arm64AppId = "{{D1ACE434-89C5-48D1-88D3-E2991DF85475}";
+  win32x64UserAppId = "{{CC6B787D-37A0-49E8-AE24-8559A032BE0C}";
+  win32arm64UserAppId = "{{3AEBF0C8-F733-4AD4-BADE-FDB816D53D7B}";
+  win32AppUserModelId = "Microsoft.CodeOSS";
+  win32ShellNameShort = "C&ode - OSS";
+  win32TunnelServiceMutex = "vscodeoss-tunnelservice";
+  win32TunnelMutex = "vscodeoss-tunnel";
+  darwinBundleIdentifier = "com.visualstudio.code.oss";
+  linuxIconName = "code-oss";
+  licenseFileName = "LICENSE.txt";
+  reportIssueUrl = "https://github.com/microsoft/vscode/issues/new";
+  nodejsRepository = "https://nodejs.org";
+  urlProtocol = "code-oss";
```

Comment:
Doesn't this break erb links from eg gitlab?

Reply:
I don't know what's the meaning of this field actually, basically reading some comments from https://github.com/microsoft/vscode-docs/issues/5215. As per my understanding, it seems to be OK with `code-oss`, right?

Reply:
I can attest that this does break some extensions. Please override it to make it `vscode`.